### ISSUE TITLE
Latest git version is broken without explicitly setting features

### ISSUE THREAD

**Issue by dfaust** [JelteF/derive_more#255]
Adding `derive_more` like this:

`derive_more = { git = "https://github.com/JelteF/derive_more", rev = "65bab87300a053df05d34c6bf73f65c08586c978" }`

causes the following error message:

`at least one derive feature must be enabled (or the "full" one enabling all the derives)`

Explicitly setting a feature like `full` fixes the issue.

**Comment by JelteF** [comment]
Yes, this is on purpose. See for the discussion on this https://github.com/JelteF/derive_more/issues/210
